### PR TITLE
algebra: adding tests for `vec_len` function

### PR DESCRIPTION
**Pull request by PNieck**
JIRA: PP-71

## Description

This PR contains tests for `vec_len` function.

## Motivation and Context

More precise tests.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## How Has This Been Tested?

- [ ] Already covered by automatic testing.
- [ ] New test added: (add PR link here).
- [x] Tested by hand on: (list targets here). `armv7a9-zynq7000-qemu`

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing linter checks and tests passed.
- [x] My changes generate no new compilation warnings for any of the targets.

## Special treatment

- [ ] This PR needs additional PRs to work (list the PRs, preferably in merge-order).
- [x] I will merge this PR by myself when appropriate.

### CODE REVIEW

File: algebra/tests/vec/various.c [hunk 1]
```diff
@@ -413,3 +413,46 @@ TEST_GROUP_RUNNER(group_vec_cross)
...
+
+TEST_TEAR_DOWN(group_vec_len)
+{
+}
+
+
+TEST(group_vec_len, vec_len_std)
+{
+	vec_t A = V1;
+	float exp = sqrt(A.x * A.x + A.y * A.y + A.z * A.z);
```
Comment (niewim19) [inline algebra/tests/vec/various.c:439]: Please use other name for `exp` such as `expected` to not confuse with `math.h` function, Applies in the other test as well

**Review by niewim19 — CHANGES_REQUESTED** [review]
Please add a test for zero length vectors

**Review by niewim19 — APPROVED** [review]
LGTM
